api-admin/src/test/java/bio/terra/pearl/api/admin/AuthAnnotationSpec.java

```diff
@@ -56,4 +56,8 @@ public static AuthAnnotationSpec withOtherAnnotations(
       List<Class<? extends Annotation>> otherAnnotations) {
     return new AuthAnnotationSpec(null, null, otherAnnotations);
   }
+
```

Comment:
This works, but I'd consider adding an `@Public` or `@Unrestricted` annotation.  That way it's clearer when reading the service that the lack of permissions is intentional.  (The annotations won't actually do anything yet, but might be useful later for logging purposes)